paasta_tools/kubernetes_tools.py

```diff
@@ -2044,6 +2047,47 @@ def get_pod_management_policy(self) -> str:
         """Get sts pod_management_policy from config, default to 'OrderedReady'"""
         return self.config_dict.get("pod_management_policy", "OrderedReady")
 
+    def format_kubernetes_job(
+        self,
+        job_label: str,
+        deadline_seconds: int = 3600,
```

Comment:
just curious: we'd talked about limiting the max uptime for the sandbox jobs - given that this doesn't do any limiting, I assume that we'll have the thresholds set in the new remote-run command? or do we want to add a max uptime check/clamp here?

Reply:
yes, I was planning to clamp the value in the API view